jvb/rootfs/defaults/jvb.conf

```diff
@@ -9,6 +10,11 @@
 {{ $JVB_ADVERTISE_PRIVATE_CANDIDATES := .Env.JVB_ADVERTISE_PRIVATE_CANDIDATES | default "true" | toBool -}}
 {{ $JVB_ADVERTISE_IPS := .Env.JVB_ADVERTISE_IPS | default "" -}}
 {{ $JVB_IPS := splitList "," $JVB_ADVERTISE_IPS -}}
+{{ $JVB_XMPP_AUTH_DOMAIN := .Env.JVB_XMPP_AUTH_DOMAIN | default "auth.jvb.meet.jitsi" -}}
```

Comment:
In a hypothetical setup where we also enable the 2nd Prosody here, wouldn't we go for jvb-*.meet.jitsi domain names?